### DISCUSSION TITLE
Nested Column Header in DataGrid?

### DISCUSSION THREAD

**Discussion by j3rmz87** [AvaloniaUI/Avalonia#8607] [Q&A]
Is it possible to created nested column headers in datagrid?

![image](https://user-images.githubusercontent.com/4407306/181190229-59ad683a-3261-4afa-ad03-cfe866b04477.png)

**Comment by maxkatz6** [comment]
There is no proper API for that, but you can try to replace [DataGridColumnHeader](https://github.com/AvaloniaUI/Avalonia/blob/0dca4bebc17e0cf640f066a4c1a4e6cc0f3b1dc1/src/Avalonia.Controls.DataGrid/Themes/Fluent.xaml#L154) template with something that will understand your use cases and your data models.
